site/documentation/content/admin-guide/jdbc-device-registry-config.md

```diff
@@ -15,10 +15,10 @@ The application is implemented as a Spring Boot application, and it uses a JDBC
 provides the following features:
 
 * Run only the registration and credentials service, or run including the tenant service.
-* By default, supports H2 and PostgreSQL
+* By default, supports H2, PostgreSQL and SQL Server
```

Comment:
I would explicitly name it MS SQL Server … as "SQL server" is a technology and "MS SQL server" is a product.

Reply:
ok, I'll change that.